### PR TITLE
Fix nil and empty string issue

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

https://github.com/wantedly/developers-account-mapper/pull/7 causes build failure

## WHAT

Fix problem in user model and register command

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sudo: required
+language: go
+go:
+  - '1.7'
+install:
+  - go get -v github.com/Masterminds/glide
+  - cd $GOPATH/src/github.com/Masterminds/glide && git checkout b5e17dc5b955dfa19539f84a53a30b9f61f2000a && go install && cd -
+  - glide install
```
Comment (dtan4) [inline .travis.yml:11]: Please use apt package.
https://github.com/dtan4/s3url/blob/3f919413dd2598fc5df23c72a820182ad2b6d814/.travis.yml#L4-L8

**Review by dtan4 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (potsbo) [reply .travis.yml:11]: @dtan4 Instruction followed, thanks

**Comment by potsbo** [comment]
I think it's ready

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍 Thank you for setting CI environment :octocat:

**Pull request merged**